docker/scripts/wpr/start.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+google-chrome --version
+
+echo 'Use Browsertime with WebPageReplay. Chrome only at the moment'
+
+RUNS="${RUNS:-5}"
+LATENCY=${LATENCY:-100}
+
+webpagereplaywrapper record --start --path /root/go/src/github.com/catapult-project/catapult/web_page_replay_go
+
+/usr/src/app/bin/browsertime.js --chrome.args host-resolver-rules="MAP *:80 127.0.0.1:8080,MAP *:443 127.0.0.1:8081,EXCLUDE localhost" -n 1 --pageCompleteCheck "return true;" "$@"
```

Comment:
Since this one is a throw away maybe just write this to /tmp with a hardcoded `--resultDir /tmp `?

Reply:
yes but if you add resultDir in the input when you run it (you probably want to have it in a specific dir so you can do something with the metrics) we will have an array of dirs.

Reply:
the same for example if you add -n 5 ... it's not a perfect solution right now.

Reply:
Ah I see now! I'll try some stuff out and see if I find any other ways.